packages/browser/src/login/oidc/redirectHandler/AuthCodeRedirectHandler.ts

```diff
@@ -215,10 +215,6 @@ export class AuthCodeRedirectHandler implements IRedirectHandler {
     await this.storageUtility.setForUser(
       storedSessionId,
       {
-        // TODO: We need a PR to oidc-client-js to add parsing of the
-        //  refresh_token from the redirect URL.
-        refreshToken:
-          "<Refresh token that *is* coming back in the redirect URL is not yet being parsed and provided by oidc-client-js in it's response object>",
```

Comment:
We're no longer storing this long-term right?

Reply:
Exactly, the refresh token is only kept in the fetch function's closure in the browser module, because since it allows getting an access token, it is as critical.

Reply:
SessionManager still persists the refresh token: https://github.com/inrupt/solid-client-authn-js/blob/73b077e08cd762496aafbb937b50572ecb76ae8d/packages/browser/src/sessionInfo/SessionInfoManager.ts#L160

Reply:
Good catch. I confirm it's not stored, it's only injected into the fetch closure there: https://github.com/inrupt/solid-client-authn-js/blob/1dd3ea0d50d0d3f8d1787ce8e826d179e319f472/packages/browser/src/login/oidc/redirectHandler/AuthCodeRedirectHandler.ts#L210. But we still look it up in storage, which is wrong, I'll remove this as part of the PR too.